### PR TITLE
eslint produces errors accoding to its smart rules [ci skip]

### PR DESCRIPTION
**Pull request by olegtaranenko**
Technical patch to eliminate annoying red marks working with test.number.ts file under WebStorm :-)

### CODE REVIEW

File: ts/src/test/base/functions/test.number.ts [hunk 1]
```diff
@@ -386,4 +386,5 @@ assert (Precise.stringLe ('3.1415', '3.14150000000000000000001'));
 // $this->assertSame (0,   Exchange::sum ());
 // $this->assertSame (2,   Exchange::sum (2));
 // $this->assertSame (432, Exchange::sum (2, 30, 400));
+// eslint-disable-next-line eol-last
```
Comment (carlosmiei) [inline ts/src/test/base/functions/test.number.ts:389]: @olegtaranenko can we add a new line at the end of the file instead?

File: ts/src/test/base/functions/test.number.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // @ts-nocheck
-// eslint-disable-next-line
-import { functions } from '../../../../ccxt.js';
-import Precise from '../../../base/Precise.js'
 import assert from 'assert';
```
[the remark this answers is not in the document]
Comment (olegtaranenko) [reply ts/src/test/base/functions/test.number.ts:2]: will do it

File: ts/src/test/base/functions/test.number.ts (hunk 1 above)
Comment (olegtaranenko) [reply ts/src/test/base/functions/test.number.ts:389]: I tried, but got error from Python.
try yourself, or we need to tune python lint or whatever

File: ts/src/test/base/functions/test.number.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (carlosmiei) [reply ts/src/test/base/functions/test.number.ts:2]: @olegtaranenko I will check don't worry

**Comment by olegtaranenko** [comment]
image
@carlosmiei 
empty line after `import {functions} ... `is required

**Comment by carlosmiei** [comment]
> image @carlosmiei empty line after `import Precise ... `is required

arghhh, @olegtaranenko can we add an eslint comment to disable that error?

**Comment by olegtaranenko** [comment]
> > image @carlosmiei empty line after `import Precise ... `is required
> 
> arghhh, @olegtaranenko can we add an eslint comment to disable that error?

yes, we can ©

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**